src/main/java/seedu/address/model/tutorial/Tutorial.java

```diff
@@ -93,7 +93,7 @@ public Tutorial(TutorialName name, Venue v, Day d, Time t, int weeks, FilteredLi
      */
     public Tutorial(TutorialName name, Venue v, Day d, Time t, int weeks, AttendanceList attendance,
                     AssessmentResultsList results) {
-        requireAllNonNull(name, d, t, results);
+        requireAllNonNull(name, d, t, weeks, results);
```

Comment:
thanks for adding checks for the other parameteres! should we check for attendance too?

Reply:
Ahh right will add it!